data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/AbstractDataBackedRestAPIIntegrationTest.java

```diff
@@ -300,7 +301,7 @@ protected Response getResponse(final String route, Map<String, String> getParams
   }
 
   protected Response post(final String route, final String postData) throws IOException {
-    final RequestBody body = RequestBody.create(JSON, postData);
+    final RequestBody body = RequestBody.create(postData, JSON);
```

Comment:
Seems that they swapped the parameters. Got this from: https://stackoverflow.com/a/57495696